src/models/v2/schema.ts

```diff
@@ -267,6 +269,10 @@ export class Schema extends BaseModel<v2.AsyncAPISchemaObject, { id?: string, pa
     return this._json.required;
   }
 
+  schemaFormat(): string {
+    return this._meta.schemaFormat || getDefaultSchemaFormat(this._meta.asyncapi.semver.version);
+  }
```

Comment:
We can even has `schemaFormat` method to recognize if user define schemaFormat field or not (default one as fallback). If is defined, it should return true, otherwise false.

Reply:
Why this would be needed? I see this is simple enough isn't?

Reply:
How you can recognize if someone define schema? 😄 I mean, it's not needed, we can add it later, but I think there can be cases to support that.

Reply:
> We can even has schemaFormat method to recognize if user defined schemaFormat field or not (default one as fallback). If is defined, it should return true, otherwise false.

But you suggest to add a `hasSchemaFormat()` method?

Reply:
Yes :)

Reply:
We had a discussion about this in the other PR and we realized it wasn't necessary. That's why the signature of `schemaFormat()` method returns always a `string` because we assume there is always a format needed even though the user didn't explicitly added.
I can't find any use case for supporting what you suggest here.

Reply:
Dont really see a reason to check whether a schemaFormat is defaulting 🤨 I would probably say let a feature request decide if that's needed 😄 

As you said, we can easily add it later 👍